packages/react/src/components/Authenticator/FederatedIdentity/controls/IdentityControl.tsx

```diff
@@ -0,0 +1,15 @@
+import { FederatedIdentityElements } from '../context/elements';
+
+export interface IdentityControl<
+  T extends Partial<FederatedIdentityElements> = FederatedIdentityElements,
+  K extends string = string,
+> {
+  (props: { provider?: K; children?: React.ReactNode }): JSX.Element;
```

Comment:
Don't quite understand what `provider` means here or how it maps to line 12. Can you explain please?

Reply:
I will change the name to providerName, and remove the Provider attribute in accordance with our new composability style. The providerName is the string passed in by the customer that maps to the providerData from context.